purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -56,8 +52,8 @@ class PostingTransactionsTests {
 
     private val attributesToMarkAsSyncSlot = slot<Map<String, SubscriberAttribute>>()
     private val attributesErrorsSlot = slot<List<SubscriberAttributeError>>()
-    private val postedProductInfoSlot = slot<ReceiptInfo>()
-    private val postedStoreUserIdSlot = slot<String>()
+    private val postedReceiptInfoSlot = slot<ReceiptInfo>()
+    private val postedStoreUserIdSlot = mutableListOf<String?>()
```

Comment:
that's very interesting! I didn't know about this

Reply:
great point, let me see

Reply:
i did a quick test, and it appears to clear itself. but i also couldn't find documentation confirming that, so i'll clear it out to be sure 🤷‍♀️